modules/util/css-builder/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay CSS Builder
 Bundle-SymbolicName: com.liferay.css.builder
-Bundle-Version: 1.1.4
+Bundle-Version: 1.2.0
 Export-Package: com.liferay.css.builder
 Main-Class: com.liferay.css.builder.CSSBuilder
 -includeresource:\
```

Comment:
If you're not using `ArgumentsUtil` anymore, you should remove it from here, and also remove all the classes added here only because `ArgumentsUtil` needs them.